tools/run_tests/run_tests.py

```diff
@@ -14,8 +14,6 @@
 # limitations under the License.
 """Run tests in parallel."""
 
-from __future__ import print_function
```

Comment:
We still need this line for Python 2, in case of printing multiple object like line 451.

Reply:
Good catch! This line must have gotten pulled out by `2to3`

Reply:
@lidizheng Addressed! Ready for re-review.